.github/workflows/devtools.yml

```diff
@@ -177,10 +177,10 @@ jobs:
       # Disabled because normal Chrome usage makes DevTools not function on these poorly constructed pages
       #   errors-expired-ssl errors-infinite-loop
       # Disabled because Chrome will follow the redirect first, and Lighthouse will only ever see/run the final URL.
-      #   redirects-client-paint-server redirects-multiple-server redirects-single-server redirects-single-client
+      #   redirects-client-paint-server redirects-multiple-server redirects-single-server redirects-single-client redirects-hash
```

Comment:
Perhaps our test definitions should begin to declare themselves with `excludeRunners`.

Reply:
@brendankenny

Reply:
Otherwise local usage of the runner will fail unless you remember to disable these tests :)